src/main/java/frc/robot/RobotContainer.java

```diff
@@ -97,7 +100,7 @@ public void updateVisionSim() {
     //            .withDriveRequestType(SwerveModule.DriveRequestType.OpenLoopVoltage);
 
     // Controller
-    private final CommandXboxController controller = new CommandXboxController(Constants.PRIMARY_CONTROLLER_PORT);
+    private final CommandXboxController controller = new CommandXboxController(0);
```

Comment:
why aren't you using the constant port?

```suggestion
    private final CommandXboxController controller = new CommandXboxController(Constants.PRIMARY_CONTROLLER_PORT);
```